instance/management/commands/instance_statistics_csv.py

```diff
@@ -244,27 +279,35 @@ def log_line(line):
     def collect_instance_statistics(
             self,
             out,
-            domain_name,
+            domain_names,
             start_date,
             end_date
-    ):  # pylint: disable=too-many-locals
+    ):
         """Generate the instance statistics CSV."""
-        instance = self.get_instance_from_domain_name(domain_name)
-        appserver = instance.get_active_appservers().first()
-        name_prefix = appserver.server_name_prefix
+        instances = self.get_instances_from_domain_names(domain_names)
+        instance_metadata = self.get_instance_metadata(instances)
 
         self.stderr.write(self.style.SUCCESS('Running playbook...'))
 
         with ansible.create_temp_dir() as playbook_output_dir:
-            self.get_elasticsearch_hits_data_summary(playbook_output_dir, name_prefix, start_date, end_date)
-            self.get_instance_usage_data(playbook_output_dir, name_prefix, appserver.server.public_ip)
+            self.get_elasticsearch_hits_data_summary(
+                playbook_output_dir,
+                instance_metadata.keys(),
+                start_date,
+                end_date
+            )
+            self.get_instance_usage_data(
+                playbook_output_dir,
+                instance_metadata
+            )
 
             csv_writer = csv.writer(out, quoting=csv.QUOTE_NONNUMERIC)
             csv_writer.writerow([
                 'Fully Qualified Domain Name',
                 'Server Name Prefix',
```

Comment:
@pcockwell, I think this is useful only inside this task and not useful in any report.